homeassistant/components/huisbaasje/const.py

```diff
@@ -0,0 +1,33 @@
+"""Constants for the Huisbaasje integration."""
+from huisbaasje.const import (
+    SOURCE_TYPE_ELECTRICITY,
+    SOURCE_TYPE_ELECTRICITY_IN,
+    SOURCE_TYPE_ELECTRICITY_IN_LOW,
+    SOURCE_TYPE_ELECTRICITY_OUT,
+    SOURCE_TYPE_ELECTRICITY_OUT_LOW,
+    SOURCE_TYPE_GAS,
+)
+
+from homeassistant.const import TIME_HOURS, VOLUME_CUBIC_METERS
+
+CONF_USERNAME = "username"
+CONF_PASSWORD = "password"
```

Comment:
These two strings are available in `homeassistant.const`, so better import from there in the file where you are using them.